contrib/gcppubsub/cmd/receive_adapter/main.go

```diff
@@ -61,7 +61,7 @@ func main() {
 
 	var env envConfig
 	if err := envconfig.Process("", &env); err != nil {
-		log.Fatal("Failed to process env var", zap.Error(err))
+		logger.Fatal("Failed to process env var", zap.Error(err))
```

Comment:
Is this a sugared logger? If so, it should be `Fatalw` (my preference is just switch to a desugared logger to make sure these bugs don't occur).

Reply:
I don't think it's a sugared one, as I cannot Desugar it. It has the Sugar() method, which makes me think is not sugared?

`logger, err := logCfg.Build()` creates a sugared one?

Reply:
Ah, my mistake. I read `log.Fatalf` above and thought it was the same logger.